<launch>
  <include file="$(find-pkg-share nuturtle_description)/launch/load_one.launch.py">
    <arg name="use_rviz" value="false" />
    <arg name="color" value="red" />
  </include>

  <node pkg="nusim" exec="nusim" name="nusim">
    <param from="$(find-pkg-share nusim)/config/basic_world.yaml"/>
    <param from="$(find-pkg-share nuturtle_description)/config/diff_params.yaml"/>
    <param name = "draw_only" value = "false" />
  </node>

  <node pkg="rviz2" exec="rviz2" args="-d $(find-pkg-share nusim)/config/nusim.rviz"/>

</launch>